.github/workflows/perf-check.yml

```diff
@@ -0,0 +1,36 @@
+name: Perf check
+
+on:
+  push:
+    branches:
+      - rc-v*
+  schedule:
+    - cron: '5 8 * * *'
+  workflow_dispatch:
+
+jobs:
+  perf-check:
+    runs-on: ubuntu-18.04
```

Comment:
@nwt: No, it doesn't explicitly require it. I guess it's down to personal preference. My philosophy on dependencies is to lock in a specific version that I've seen working whenever possible to avoid unplanned breakages. That is, if it's set to "latest", GitHub might advance the version on their Runners to a new "latest" without warning and cause an unplanned breakage at an inconvenient time. If I lock in to a specific version that I've seen working, I have more hope that I'd see GitHub's bulletins of "we're phasing out ubuntu 18.04!" so I can test and advance the pointer to a newer explicit version as part of a more carefully-planned exercise.

All that said, I don't expect Ubuntu versions to cause a breakage here, so I'm happy to advance it to make it look like the others.

Reply:
Please do. Convention trumps personal preference.